### DISCUSSION TITLE
How to make ecode portable?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#135] [General]
With Lite XL, the only thing you have to do is:

> To make Lite XL portable (e.g. running Lite XL from a thumb drive), simply create a `user` folder where `lite-xl.exe` is located. Lite XL will load and store all your configurations and plugins in the folder.

https://github.com/lite-xl/lite-xl/blob/master/README.md

**Comment by emeraldtip** [comment]
> Portable mode can be invoked by command line argument with --portable or -p or by creating a file besides the ecode binary called portable_mode.txt.
